### ISSUE TITLE
Ability to set the hardware time clock for each VM

### ISSUE THREAD

**Issue by phirestalker** [utmapp/UTM#3124]
I run some Linux VMs which expect the hardware clock to be in UTC, but my host is MacOS so it is not. This makes certain security software (such as tor) not function properly.

I would like to be able to choose whether the VM should use UTC or the local timezone for the hardware clock. I know there are options in Qemu for this, so I guess this is mostly an interface upgrade.

Thanks for the great software! I thought I would never get to run virtual machines on my M1 mac, but your software makes it easy.

**Comment by ViViDboarder** [comment]
I've got the same issue. My VM is showing the correct time zone, but it appears that it's assumed the hardware clock is UTC as well so I'm skewed by my timezone offset.

**Comment by ViViDboarder** [comment]
I managed to get the time working by adding additional QEMU flags. I went to machine preferences, QEMU, and then added `-rtc base=utc,clock=host` at the bottom.

**Comment by adespoton** [comment]
I'm not getting that to work; I attempted to add -rtc time="2001-10-01",clock=vm and I got an error that I needed to use the format "2006-06-17" -- which is the format I'm using.  It seems to be getting confused between this flag and the already existing -rtc flag just prior.

Maybe the custom page should also let us remove or mask existing flags set from elsewhere in the UI?  This would also let me fix things like a USB bus being presented to CUDA systems (which don't have a USB bus, or any way to address one).
